RigidAlignment.xml.in

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <executable>
+@MODULE_CATEGORY_CONFIG@
 <title>RigidAlignment</title>
```

Comment:
Should we add an option for this as well just for consistency? I'm fine either way, since SALT doesn't modify it.

Reply:
Thanks for the review.

re: title

Since I think  the title matches the name of the executable, I didn't include the customization  here as having both being different may break some assumption within Slicer.  I may be wrong ... I just didn't have a chance to look into it and preferred to be conservative.

Generally speaking, we should discuss with the other dev how to override categorization (and the title for that matter) for any modules.